### ISSUE TITLE
Missing docker's load

### ISSUE THREAD

**Issue by mynote** [mariotoffia/FluentDocker#288]
I'm not sure if i am wrong, but I couldn't find a wrapper for docker's load functionality for offline usage.

Example:  
docker save --output myimage.tar myimage  
docker load --input myimage.tar

**Comment by mariotoffia** [comment]
@mynote That is true! If you'd like, please submit a PR that implements those commands (Commands/client.cs) - around this fashion:

```cs
    public static CommandResponse<IList<string>> Load(this DockerUri host, string image, bool quiet = true, ICertificatePaths certificates = null)
    {
      var args = $"{host.RenderBaseArgs(certificates)}";
      var options = string.Empty;

     if (quiet) {
      options += "--quiet";
    }

      return
        new ProcessExecutor<StringListResponseParser, IList<string>>(
          "docker".ResolveBinary(),
          $"{args} load --input {image} {options}").Execute();
    }
```

...and a unit test to save and then load the image...

Cheers,
 Mario

**Comment by mynote** [comment]
Thank you, Mario. Cheers, Emanuel

**Comment by mariotoffia** [comment]
**Resolved in v3.0.0.** The v3 ground-up rewrite implements (or fixes) what this issue asked for.

This is a feature request asking for wrappers around `docker save --output myimage.tar` and `docker load --input myimage.tar` for offline image transfer. The maintainer (mariotoffia) accepted it in the thread and sketched a `Load` helper. In the v3 rewrite this is fully implemented: the `IImageDriver` interface declares `SaveAsync`, `LoadAsync`, and `ImportAsync`, and all three concrete drivers implement them (Docker CLI via `docker save -o`/`docker load -i`/`docker import`, Docker API via the streaming endpoints, and Podman CLI via `save -o`/`load -i`). The capability is also surfaced at the service layer via `IImageService.SaveAsync`, and there are unit tests plus integration tests covering a real save-then-load round-trip. The requested functionality therefore exists and the issue should be closed.

<sub>Where this lives in v3: Interface: FluentDocker/Drivers/IImageDriver.cs — region "Save/Load/Import Operations" declares SaveAsync (lines 163-167), LoadAsync (176-179), ImportAsync (191-197). Docker CLI impl: FluentDocker/Drivers/Docker/Cli/Components/DockerCliImageDriver.Operations.cs — SaveAsync runs `save -o "<path>" <images>` (line 139), LoadAsync runs `load -i "<path>"` and parses "Loaded image:" output (lines 159-202), ImportAsync runs `import` (lines 205-243). Docker API impl: FluentDocker/Drivers/Docker/Api/Components/DockerApiImageDriver.cs — SaveAsync (line 234), LoadAsync (line 265), ImportAsync (line 311). Podman CLI impl: FluentDocker/Drivers/Podman/Cli/Components/PodmanCliImageDriver.Operations.cs —…</sub>

Closing as part of the v3 issue sweep. If your scenario still isn't covered by v3, please reopen with a v3 reproduction and we'll take another look. Thanks for the report! 🙏
